File: payment_methods/dotnet/get_payment_methods.cs
using mercadopago;
using System;
using System.Collections;

public class MyClass {
	public static void Main () {
		MP mp = new MP("ACCESS_TOKEN");

		Hashtable paymentMethods = mp.get ("/v1/payment_methods");

		Console.WriteLine (paymentMethods.ToString());
	}
}

File: payment_methods/java/get_payment_methods.java
import com.mercadopago.MP;
import org.codehaus.jettison.json.JSONObject;

public class MyClass {
	public static void main (String args[]) {
		MP mp = new MP ("ACCESS_TOKEN");

		JSONObject paymentMethods = mp.get ("/v1/payment_methods");

		System.out.println(paymentMethods.toString());
	}
}
